Source/WebCore/platform/graphics/gstreamer/AudioTrackPrivateGStreamer.cpp

```diff
@@ -51,6 +51,27 @@ AudioTrackPrivateGStreamer::AudioTrackPrivateGStreamer(WeakPtr<MediaPlayerPrivat
     , m_player(player)
 {
     ensureDebugCategoryInitialized();
+
+    if (!m_pad)
+        return;
+
+    g_signal_connect_swapped(m_pad.get(), "notify::caps", G_CALLBACK(+[](AudioTrackPrivateGStreamer* track) {
```

Comment:
I wonder if it would be possible to de-duplicate these signal handlers? Right now they're almost copy/paste in audio and video tracks.